qiskit_experiments/framework/json.py

```diff
@@ -555,6 +563,11 @@ def object_hook(self, obj):
                 return _deserialize_bytes(obj_val)
             if obj_type == "set":
                 return set(obj_val)
+            if obj_type == "Gate":
```

Comment:
I think it's safer to assign more specific type notation such as `SingleGateCircuit` (probably there is better name). If QPY exposes interface of gate serialization in future we may switch to that API. However, this will block that upgrade because there will be conflict of representation under the same type name. If you have unique name for this hack, we can just deprecate it.

Reply:
It should just be "Instruction", which basically is qiskits notion of "single gate circuit" (though in reality it could have an arbitrary circuit in its definition).

Reply:
See here: [5c93061](https://github.com/Qiskit/qiskit-experiments/pull/684/commits/5c93061706c68ecdbc87c0767597002192df46c1)